env/archivist_testnet.env

```diff
@@ -0,0 +1,20 @@
+NAME=pete_testing
+BASE_PATH=/data
+BOOT_NODES=/dns4/bootnode-eu-central-1-0.test.azero.dev/tcp/30333/p2p/12D3KooWRkGLz4YbVmrsWK75VjFTs8NvaBu42xhAmQaP4KeJpw1L/dns4/bootnode-eu-west-1-0.test.azero.dev/tcp/30333/p2p/12D3KooWFVXnvJdPuGnGYMPn5qLQAQYwmRBgo6SmEQsKZSrDoo2k/dns4/bootnode-eu-west-2-0.test.azero.dev/tcp/30333/p2p/12D3KooWAkqYFFKMEJn6fnPjYnbuBBsBZq6fRFJZYR6rxnuCZWCC/dns4/bootnode-us-east-1-0.test.azero.dev/tcp/30333/p2p/12D3KooWQFkkFr5aM5anGEiUCQiGUdRyWgrdpvSjBgWAUS9srLE4/dns4/bootnode-us-east-2-0.test.azero.dev/tcp/30333/p2p/12D3KooWD5s2dkifJua69RbLwEREDdJjsNHvavNRGxdCvzhoeaLc
+CHAIN=/data/testnet_chainspec.json
+NODE_KEY_PATH=/data/p2p_secret
+LOG_FILE=/data/aleph-node.log
+POOL_LIMIT=100
+UNIT_CREATION_DELAY=300
+PORT=30333
+RPC_PORT=9944
+METRICS_PORT=9615
+RUST_LOG=info
+SYNC=Fast
```

Comment:
This env is not part of https://github.com/Cardinal-Cryptography/aleph-node/blob/main/docker/docker_entrypoint.sh, so it is internal to the runner file. But in the runner file, `SYNC` can only have `true` value, when `--sync-from-genesis` is passed , so is `Fast` correct value here?

Btw it would be nice to have some grouping or even naming convention for envs in this file, to have two groups: ones used internally by runner (like `SYNC`) and ones used by https://github.com/Cardinal-Cryptography/aleph-node/blob/main/docker/docker_entrypoint.sh (like `RPC_PORT`).

Reply:
`SYNC` being an internal param of the script is an unfortunate name clash that is solved in a commit I'll push in a second. Otherwise, I agree of course but the config used here is intended to be passed directly (well, after variable expansion) to the docker: if the runner uses these values in the script it's because e.g. we need to set the port mappings. (so no param in this file should be internal only)